### DISCUSSION TITLE
False positive for GHSA-gmc6-fwg3-75m5 on MimeKitLite?

### DISCUSSION THREAD

**Discussion by pjanotti** [jstedfast/MimeKit#1054] [General]
Hi, both `docker scout` and `grype` are reporting https://github.com/advisories/GHSA-gmc6-fwg3-75m5 on `mcr.microsoft.com/windows/servercore:ltsc2022` Docker image, the file being:
```terminal
/Files/Program Files/Windows Defender Advanced Threat Protection/Classification/Dprt/MimeKitLite.dll
```
per my reading of:

* https://github.com/advisories/GHSA-447r-wph3-92pm
* https://github.com/jstedfast/MimeKit/commit/aef4eda75525848b992ce5e1f9b87399000fffb6

This is a false positive. Could you please confirm if that is really the case?

**Comment by jstedfast** [comment]
Yes, it is a false reading. It only affects MimeKit (I published a release of MimeKitLite, though, just to keep versions identical)
